test/test_t_test.cpp

```diff
@@ -105,5 +152,23 @@ int main()
     test_integer<int32_t>();
     test_integer<int64_t>();
     test_integer<uint32_t>();
+    test_two_sample_t<float>();
+    test_two_sample_t<double>();
```

Comment:
Could you also test compatibility with `boost::multiprecision`? Quad should be fine.